src/algebra/group/pi.lean

```diff
@@ -165,6 +165,17 @@ def monoid_hom.coe_fn (α β : Type*) [mul_one_class α] [comm_monoid β] : (α
   map_one' := rfl,
   map_mul' := λ x y, rfl, }
 
+/-- Monoid homomorphism between the function spaces `I → α` and `I → β`, induced by a monoid
+homomorphism `f` between `α` and `β`. -/
+@[to_additive "Additive monoid homomorphism between the function spaces `I → α` and `I → β`,
+induced by an additive monoid homomorphism `f` between `α` and `β`", simps]
+def monoid_hom.comp_left {α β : Type*} [mul_one_class α] [mul_one_class β] (f : α →* β)
```

Comment:
I think this should be called `pi.comp_left_monoid_hom` to match `pi.eval_monoid_hom`, and to disambiguate from `monoid_hom.comp_hom`

Reply:
Oh nevermind, I'd missed the first argument was a bundled hom